### ISSUE TITLE
Back button on World Map goes back to Main Menu

### ISSUE THREAD

**Issue by Surram** [LewisRFlanagan/MyFirstRepository#6]
Pressing the back button while on the world map takes you back to the main menu (ie. where the New Game button is). Progress is lost.

**Comment by LewisRFlanagan** [comment]
Yes. If you exit the game without saving, your progress will be lost. We
should probably add a warning message that lets the user go back or an
autosave point, unless you have a better idea.
